### PR TITLE
Card variants fixes and additional card variants

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
## New card Variant
image

## Add new animation when the card hovered
[Card CodeSandBox Link](https://qg4h2.csb.app/pages/card/card.html)

### CODE REVIEW

File: pages/card/card.css
Comment (gadgetfather) [inline pages/card/card.css:21]: You can remove this comment before merging

File: pages/card/card.html
Comment (gadgetfather) [inline pages/card/card.html:25]: Also while hovering on this icon you can make its color red

**Review by gadgetfather — APPROVED** [review]
Clean code. PR is good to merge

**Review by kart-c — APPROVED** [review]

File: pages/card/card.css
Comment (ruchiragoswami) [inline pages/card/card.css:14]: Just want to draw your attention, the color is hard-coded. Not sure if you want it that way or in a variable.

**Review by ruchiragoswami — APPROVED** [review]
Great work Kevin ! 
Just a general comment, colors are hard-coded at some places. Not sure if you want to keep it that way or in a variable.

File: pages/card/card.html
Comment (Kevin-Solomon) [reply pages/card/card.html:25]: Yes but only the border color of the heart changes to red.
To make the actual heart fill with red we need to add a new class name.

File: pages/card/card.css
Comment (Kevin-Solomon) [reply pages/card/card.css:14]: This is just a temporary color. 
I will change this once i get approval on the CL to use its primary color variable.

**Pull request merged**
